src/caf/space/weighted_funcs.py

```diff
@@ -4,21 +4,23 @@
 These are called in the 'weighted_trans' method in ZoneTranslation.
 """
 ##### IMPORTS #####
+from typing import Optional
 import logging
 import warnings
 from functools import reduce
 import pandas as pd
 
+
 # pylint: disable=import-error
 import geopandas as gpd
 
 # pylint: enable=import-error
 
-from caf.space import inputs, zone_correspondence
+from caf.space import inputs
 
 ##### CONSTANTS #####
 logging.captureWarnings(True)
-LOG = logging.getLogger(__name__)
+LOG = logging.getLogger("SPACE")
```

Comment:
How do you expect this logger to work? I got the impression it was inherently part of the ZoneTranslation class, so would expect it to be in there?

Reply:
I was thinking I needed to create the logger in each file for it to be found when called in ZoneTranslation. Think I'm still not completely sure how logging works and got it to a point it worked then didn't touch it again!

Reply:
I've seen this pop up a few times, all with the same name. I think the module loggers need to go back to `__name__`, while the ZoneTranslation class is set to start / end a logger for the function calls it is set to log information for.